modules/ROOT/partials/toolbar-button-ids/typography-toolbar-buttons.adoc

```diff
@@ -0,0 +1,7 @@
+[cols="1,3",options="header"]
+|===
+|Toolbar button identifier |Description
+|`+typopgraphy+` | Runs the currently active set of Advanced Typography rules over the document or the current selection. Also contains a menu of available language-specific rule sets. When the menu reveal, rather than the icon proper, is pressed, this menu is displayed and a new language-specific rule set can be selected.
```

Comment:
Should we mention whether or not the typography rules are executed when changing the language?